src/unreleased.ts

```diff
@@ -7,3 +7,4 @@
 
 // Components
 export * from './ActionList2'
+export * from './NewButton'
```

Comment:
This wouldn't be able to grab the `export default Button`, only `ButtonProps` :(

Reply:
Ah! Haven't tested this properly yet.